contracts/utils/GsnEip712Library.sol

```diff
@@ -70,16 +70,16 @@ library GsnEip712Library {
                 IRelayRecipient.isTrustedForwarder.selector, relayRequest.relayData.forwarder
             )
         );
-        require(success, "isTrustedForwarder reverted");
-        require(ret.length == 32, "isTrustedForwarder returned invalid response");
+        require(success, "isTrustedForwarder: reverted");
+        require(ret.length == 32, "isTrustedForwarder: bad response");
```

Comment:
unrelated to the PR: note here how returned data is checked to exist. this is missing from the `RelayHub`'s handling of a call to `innerRelayCall`, I believe